src/client/containers/QuizResultPage/quizResultGetCareer.apiRequest.js

```diff
@@ -0,0 +1,34 @@
+async function getAnswer(id) {
+  const respons = await fetch(`/api/answers/${id}`);
+  const answer = await respons.json();
+  return answer;
+}
+async function careerSet(userId) {
+  let uxPoints = 0;
+  let fullstackPoints = 0;
+  let dataAnalystPoints = 0;
+  let career;
+
+  const response = await fetch(`/api/quiz-results?userId=${userId}`);
```

Comment:
It looks like you have two different variables called response in this file. You should rename them so that it is more clear which response is referenced.

Reply:
done